src/features/auth/useImagePreloader.tsx

```diff
@@ -26,6 +26,7 @@ import retreat from "assets/land/retreat.webp";
 import treasureIsland from "assets/land/treasure_island.webp";
 import snowKingdom from "assets/land/snow_kingdom.webp";
 import stoneHeaven from "assets/land/stone_haven.webp";
+import silhouette from "assets/bumpkins/silhouette.png";
```

Comment:
I am wondering if this silhouette could be a little more opaque. It was showing for me a lot when loading modals etc and it feels pretty dark 🤔

Reply:
Good idea - definitely open to tweaking this 😄